apps/src/javalab/Javalab.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import {getStore, registerReducers} from '../redux';
+import JavalabView from './JavalabView';
+import javalab from './javalabRedux';
+import {TestResults} from '@cdo/apps/constants';
+
+/**
+ * On small mobile devices, when in portrait orientation, we show an overlay
```

Comment:
Is Java Lab going to be supported on mobile/tablet?

Reply:
tablet yes, mobile not in pilot and I think only view-only.